### PR TITLE
Design and copy changes on “/fellowships/apply/” page

### PR DESCRIPTION
**Pull request by taisdesouzalessa**
Related to: #1195 

@gvn could you please review this code? Thanks!

### CODE REVIEW

File: network-api/networkapi/fellows/templates/fellows_apply.html [hunk 1]
```diff
@@ -10,27 +10,53 @@
...
+        </p>
+        <p>Who are Mozilla Fellows? They are working to stop the spread of misinformation, to put individuals in control of their data, and to keep artificial intelligence accountable. They are leaders who ensure smart cities and next-generation voice technology are diverse and equitable, and who conduct open research.
+        </p>
+        <p>Mozilla Fellowships are a transformative experience for emerging leaders concerned with making the internet a safer, more accessible resource for everyone. Fellows expand their network and sphere of influence; design impactful projects with the potential to reach millions; and learn from and collaborate with a global community of thousands of Mozillians. Mozilla Fellows are also awarded competitive compensation and benefits. 
+        </p>
+        <h4 class="h4-light-black">Mozilla Fellows fit three particular profiles:</h4>
+      </div><!-- end of col 12 -->
```
Comment (gvn) [inline network-api/networkapi/fellows/templates/fellows_apply.html]: Please don't add all these extra `end/beginning` comments. This is just extra noise for editors with code folding (Atom, Sublime, etc) and more to maintain to retain accuracy if the class names change.

Comment (taisdesouzalessa) [reply network-api/networkapi/fellows/templates/fellows_apply.html]: Sure! Thanks for the tip Gavin! I will change it and submit the PR again I guess?

**Comment by taisdesouzalessa** [comment]
@gvn I did the requested changes and pushed the code again. I see it in the "Commits" tab. Let me know if there is any other action item I should take? Sorry...newbie...lol!

File: network-api/networkapi/fellows/templates/fellows_apply.html [hunk 2]
```diff
@@ -10,27 +10,53 @@
...
     <div class="row">
       <div class="col-12">
         {% include 'contact_banner.html' %}
       </div>
-    </div>
-  </div>
-</div>
+    </div> 
+  </div> 
+</div>
```
Comment (gvn) [inline network-api/networkapi/fellows/templates/fellows_apply.html]: Can you remove the trailing space from these 3 lines? Also, not sure what editor you are using, but if it has a remove trailing spaces option that's best to leave turned on.

File: network-api/networkapi/fellows/templates/fellows_apply.html [hunk 3]
```diff
@@ -10,27 +10,53 @@
...
+        <p>Mozilla Fellowships provide resources, tools, community and amplification to those building a more ​humane​ ​digital​ ​world. During their tenure, Fellows use their skill sets — in technology, in activism, in science, in policy — to design products, run campaigns, influence policy and ultimately lay the groundwork for a more open and inclusive internet.
+        </p>
+        <p>Who are Mozilla Fellows? They are working to stop the spread of misinformation, to put individuals in control of their data, and to keep artificial intelligence accountable. They are leaders who ensure smart cities and next-generation voice technology are diverse and equitable, and who conduct open research.
+        </p>
+        <p>Mozilla Fellowships are a transformative experience for emerging leaders concerned with making the internet a safer, more accessible resource for everyone. Fellows expand their network and sphere of influence; design impactful projects with the potential to reach millions; and learn from and collaborate with a global community of thousands of Mozillians. Mozilla Fellows are also awarded competitive compensation and benefits. 
+        </p>
```
Comment (gvn) [inline network-api/networkapi/fellows/templates/fellows_apply.html]: Can you move these closing `</p>` tags to the previous line?

File: network-api/networkapi/fellows/templates/fellows_apply.html (hunk 2 above)
Comment (taisdesouzalessa) [reply network-api/networkapi/fellows/templates/fellows_apply.html]: I didn't even know this existed, Gavin! Thanks for the pro tip here :). Yay!

**Comment by taisdesouzalessa** [comment]
@gvn I did the changes you requested :).

**Pull request merged**